assets/tests/components/card.test.tsx

```diff
@@ -18,24 +18,24 @@ describe("Card", () => {
     expect(
       result.queryByText(/Foo/)?.parentElement?.parentElement?.parentElement
         ?.className
-    ).toMatch(/m-card--read/)
+    ).toMatch(/m-card--inactive/)
```

Comment:
suggestion(non-blocking): maybe while updating these, we could use `toHaveClass`?

Reply:
Addressed this and other test cleanup for this file in [aaed28f](https://github.com/mbta/skate/pull/1920/commits/aaed28fe45f4f0e55bbf09a5cfeb5a92f6c2b03c)

Reply:
🙌🏻 Thank you 😻